### PR TITLE
Add one more wallet saga test

### PR DESCRIPTION
**Pull request by akalin-keybase**

### CODE REVIEW

File: shared/actions/wallets.js [hunk 1]
```diff
@@ -24,7 +24,17 @@ import {RPCError} from '../util/errors'
 import {isMobile} from '../constants/platform'
 import {actionHasError} from '../util/container'
 
-const buildPayment = (state: TypedState, action: any) => {
+const buildPayment = (
+  state: TypedState,
+  action:
```
Comment (akalin-keybase) [inline shared/actions/wallets.js]: better typing here

File: shared/actions/wallets.js [hunk 2]
```diff
@@ -648,11 +658,13 @@ const acceptDisclaimer = (state: TypedState, action: WalletsGen.AcceptDisclaimer
 
 const maybeNavToLinkExisting = (state: TypedState, action: WalletsGen.AcceptDisclaimerPayload) =>
   action.payload.nextScreen === 'linkExisting' &&
-  Saga.put(Route.navigateTo(
-    isMobile
-      ? [Tabs.settingsTab, SettingsConstants.walletsTab, 'linkExisting']
-      : [Tabs.walletsTab, 'wallet', 'linkExisting']
-  ))
+  Saga.put(
```
Comment (akalin-keybase) [inline shared/actions/wallets.js]: not sure why prettier did this now

File: shared/actions/__tests__/wallets.test.js [hunk 3]
```diff
@@ -149,3 +149,41 @@ it('build and send payment', () => {
...
+  sendingIntentionXLM: false,
+  toErrMsg: '',
+  worthDescription: '21.4168160 XLM',
+  worthInfo: '$1.00 = 4.2833632 XLM\nSource: coinmarketcap.com',
+}
+
+it('build and send request', () => {
+  const {dispatch, getState} = startReduxSaga()
+  const buildRPC = jest.spyOn(RPCStellarTypes, 'localBuildRequestLocalRpcPromise')
+  buildRPC.mockImplementation(() => new Promise(resolve => resolve(buildRequestRes)))
```
Comment (buoyad) [inline shared/actions/__tests__/wallets.test.js]: you can use `Promise.resolve(buildRequestRes)` instead

Comment (akalin-keybase) [reply shared/actions/__tests__/wallets.test.js]: Cool! Also fixed a bunch of other places in the tests.

**Pull request merged**